fastcrypto/src/bls12381.rs

```diff
@@ -295,6 +298,16 @@ fn get_128bit_scalar<Rng: AllowedRng>(rng: &mut Rng) -> blst_scalar {
     }
 }
 
+fn get_one() -> blst_scalar {
+    let mut one = blst_scalar::default();
+    let mut vals = [0u64; 4];
```

Comment:
As it's not clear if blst uses little-endian, can we add a test that this is really 1?

Reply:
The API don't really allow an easy test of that, but I've replaces blst_scalar_from_uint64 with blst_scalar_from_le_bytes to make it more clear, that the encoding is little endian.

Reply:
@jonas-lj another way is to multiply a BLS elliptic curve point with that scalar, and if it was `one`, the resulted point would remain the same.

Reply:
Yea, I started writing a test that the corresponding public key is the generator until I found this solution.